electron/src/renderer/components/Simulator.vue

```diff
@@ -306,6 +306,19 @@
         </div>
       </div>
     </loading>
+    <v-dialog v-model="error" width="auto">
+      <v-card flat>
+        <v-card-title class="pa-4">
+          Error Generating Stitch Plan
+        </v-card-title>
+        <v-card-text>
+          <pre>{{ error_message }}</pre>
```

Comment:
Could we use `<p style="white-space: pre-wrap;">{{ error_message }}</p>` instead?
This would make it easier to read long error message on small screens while still obeying line breaks.